app/src/main/java/org/simple/clinic/home/patients/PatientsScreenController.kt

```diff
@@ -211,4 +211,9 @@ class PatientsScreenController @Inject constructor(
           { ui: Ui -> ui.showScanCardButton(isScanCardFeatureEnabled) }
         }
   }
+
+  private fun openScanSimpleIdCardScreen(events: Observable<UiEvent>): Observable<UiChange> =
+      events
+          .ofType<ScanCardIdButtonClicked>()
+          .map { Ui::openScanSimpleIdCardScreen }
```

Comment:
Do we even need to route clicks from this button through the controller?

Reply:
Okay, looks like you've added some logic in later commits

Reply:
Yes, I moved the permissions check to the Patients Screen.